### DISCUSSION TITLE
How to customize grouped column header?

### DISCUSSION THREAD

**Discussion by SergeyAlexeev** [Comcast/react-data-grid#3371] [General]
Hi!

I have a data grid with grouped columns and I want to customize headers.

At first, I want to setup grouped columns header **height** like below:

<img width="553" alt="Screenshot 2023-10-19 at 17 18 23" src="https://github.com/adazzle/react-data-grid/assets/1897913/18af0944-a062-4996-8b7d-0f5ef9b9ef76">

 I found the `headerRowHeight` prop, but it affects both header rows. 

And the second. I need to customize grouped cell content. I've tried to use the `renderGroupCell` and `renderHeaderCell` props, but this approach doesn't help as well.

I've prepared a small [example](https://codesandbox.io/p/sandbox/recursing-platform-hctfsl?file=%2Fsrc%2FApp.tsx%3A31%2C5-31%2C21) that demonstrates described issues. 

Does anybody know how to fix it?)

**Comment by eliasmelgaco** [comment]
I also wanted to know 🙏🏽
